types/react-notification-system/index.d.ts

```diff
@@ -70,10 +70,14 @@ declare namespace NotificationSystem {
 
     export interface Attributes {
```

Comment:
Ohh... ok.
Then you can use `ClassAttributes<System>`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6efe6e60c2a8f12453ef92df5231907b238b7e88/types/react/index.d.ts#L422